cmd/livepeer/livepeer.go

```diff
@@ -346,9 +348,12 @@ func main() {
 	}
 	if *s3bucket != "" {
 		s3bp := strings.Split(*s3bucket, "/")
-		drivers.S3REGION = s3bp[0]
 		drivers.S3BUCKET = s3bp[1]
 	}
+	if *gsBucket != "" && *gsKey == "" || *gsBucket == "" && *gsKey != "" {
```

Comment:
Couldn't this be a bit shorter like `if *gsKey == "" || *gsBucket == "" {` ?

Reply:
no, here I'm checking if only one of flags defined, and not both

Reply:
Thank you, I now understand what we're doing here. This condition is checking that if the node operator wants to use GS, they must provide both variables, and if they don't want to use GS, they should not provide either of them.